lib/notifications.dart

```diff
@@ -0,0 +1,209 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:tutorial/constants.dart';
+
+class NotificationData {
+  const NotificationData({
+    required this.timeOfDay,
+    required this.reasonOfNotification,
+  });
+
+  final String timeOfDay;
+  final String reasonOfNotification;
+}
+
+// TODO: Determine if the notifcations will be a drop-down / clickable / expandable.
+class PastNotification extends StatelessWidget {
+  const PastNotification({super.key, required this.notificationData});
+
+  final NotificationData notificationData;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: const BoxDecoration(
+        border: Border(
+          bottom: BorderSide(
+            color: Color(0xFFD4D4D4),
+            width: 1.0,
+          ),
+        ),
+      ),
+      padding: const EdgeInsets.only(bottom: 16.0),
+      child: Row(
+        children: [
+          Column(
+            children: [
+              Container(
+                decoration: BoxDecoration(
+                  shape: BoxShape.circle,
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.black.withOpacity(0.2),
+                      spreadRadius: 0,
+                      blurRadius: 5,
+                      offset: const Offset(0, 0),
+                    ),
+                  ],
+                ),
+                child: ClipOval(
+                  child: CircleAvatar(
+                    radius: 25,
+                    backgroundColor: Colors.white,
+                    child: Image.asset(
+                      "./assets/PI-Avatar.png",
+                      fit: BoxFit.cover,
+                      alignment: Alignment.center,
+                    ),
+                  ),
+                ),
+              ),
+            ],
+          ),
+          const SizedBox(
+            width: 16,
+          ),
+          Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(notificationData.timeOfDay,
+                  style: const TextStyle(
+                      color: grayColor,
+                      fontWeight: FontWeight.w500,
+                      fontSize: 12,
+                      height: 1.5)),
+              Container(
```

Comment:
wrap the column in Expanded() and I think it will solve your issue. It basically says take the remaining space in the row(column, but usually row) as its size. then the text will wrap